syft/frameworks/torch/he/fv/encryption_params.py

```diff
@@ -0,0 +1,67 @@
+from logging import warning
+
+
+class EncryptionParams:
+    """Represents user-customizable encryption scheme settings. The parameters (most
+    importantly poly_modulus, coeff_modulus, plain_modulus) significantly affect
+    the performance, capabilities, and security of the encryption scheme. Once
+    an instance of EncryptionParameters is populated with appropriate parameters,
+    it can be used to create an instance of the FVContext class, which verifies
+    the validity of the parameters, and performs necessary pre-computations.
+    """
+
+    def __init__(self):
+        self._poly_modulus_degree = 2
+        self._coeff_modulus = []
+        self._plain_modulus = 0
```

Comment:
Can't we have the init function take keyword arguments to set the attributes on instantiation?